### PR TITLE
Added dnf and yum to linux.

### PR DESCRIPTION
**Pull request by cdminigun**
Added support for Fedora, CentOS and RHEL.

### CODE REVIEW

File: pages/linux/dnf.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# dnf
+
+> RHEL, Feodra, and CentOS package management utility (Replaces yum)
```
Comment (waldyrious) [inline pages/linux/dnf.md]: Reordering this sentence makes more sense IMO: "Package management utility for RHEL, Feodra, and CentOS (replaces yum)"

File: pages/linux/yum.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# yum
+
+> RHEL, Feodra, and CentOS package management utility (For older versions)
```
Comment (waldyrious) [inline pages/linux/yum.md]: Same as above.

**Comment by waldyrious** [comment]
Just a minor nitpick, which I've added as inline comments. Other than that, looks great :+1:

**Comment by cdminigun** [comment]
Thanks for the recommendations! I agree that your wording makes more sense and I have updated the files.

**Comment by waldyrious** [comment]
Looks good to me :)

While we wait for another approval (we have implemented a policy of only merging a PR after at least 2 thumbs-up from the maintainers), feel free to fix the capitalization of "For older versions" (should be lowercase).

**Comment by cdminigun** [comment]
Great policy change! Will definitely help validate commits. 

I've gone ahead and changed the capitalization as per your request.

File: pages/linux/dnf.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# dnf
+
+>Package management utility for RHEL, Feodra, and CentOS (replaces yum)
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent dnf commands.
+
+`dnf update`
+
+- install a new package
```
Comment (felixonmars) [inline pages/linux/dnf.md]: I think the first letter of a sentence should be consistently capitalized, so this line should become "Install a new package", and same for the two examples below.

**Comment by cdminigun** [comment]
Changed were added per your request. Thanks for catching that, @felixonmars.

File: pages/linux/yum.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent yum commands.
+
+`yum update`
+
+- Install a new package
+
+`yum install {{package}}`
+
+ - Install a new package and assume yes to all questions (Also works with update, great for automated updates)
```
Comment (felixonmars) [inline pages/linux/yum.md]: And now it has different spacing here between the two files. Sorry to be nitpicking :P

**Comment by cdminigun** [comment]
I believe I fixed what you were talking about. Noticed that a few lines had spaces before text. 

If that wasn't the issue, would you please clarify? Also, no problem. I'm more than happy to help make it as perfect as possible.

**Comment by felixonmars** [comment]
All fixed now, thanks :)

**Pull request merged**